Header header
int64 no_clusters
int64 no_periods
int64 spatial_dim
float64[] periods
WHyTeMapClusterMsg[] clusters
